github/ipfs.yml

```
@@ -3621,21 +3637,18 @@ repositories:
         - JS Core Team
         - Repos - JavaScript
     visibility: public
-  js.ipfs.io:
+  js.ipfs.tech:
     branch_protection:
       master: {}
     collaborators:
       push:
         - web3-bot
     description: The Website for the JavaScript implementation of the IPFS protocol
```

The quick fix I introduced to reenable sync can't handle moving files between repos. That's why we have to "recreate" it. 
```suggestion
    description: The Website for the JavaScript implementation of the IPFS protocol
    files:
      .github/workflows/stale.yml:
        content: .github/workflows/stale.yml
        overwrite_on_create: true
```